Src/stm32f4xx_it.c

```diff
@@ -88,23 +88,23 @@ const uint8_t V2_CLOCK_LOW_A = 31;
 
 /* Private macro -------------------------------------------------------------*/
 /* USER CODE BEGIN PM */
-#define WAIT_4_CYCLES asm("ADD     R1, R2, #0\nADD     R1, R2, #0\nADD     R1, R2, #0\nADD     R1, R2, #0")
+#define WAIT_4_CYCLES __asm("ADD     R1, R2, #0\nADD     R1, R2, #0\nADD     R1, R2, #0\nADD     R1, R2, #0")

```

Comment:
Why __asm() vs asm()? Please explain. Thanks :)

Reply:
'asm' is the GNU-extension spelling of '__asm' -- the former isn't available when attempting to compile in C99/C11 mode (the default is typically C11 + GNU extensions in GCC-land), so this is effectively a low-cost move away from GNU extensions where the C-specified identifier isn't much less readable.